### PR TITLE
[CR] Stop duplicate item removal. fixes #21586.

### LINKED ISSUE #21586: Error while placing a funnel
I am using a custom compiled version ( based on beeff69 + [ChezzoSounds](https://github.com/Chezzo/Cataclysm-DDA/tree/ChezzoSounds)) .

I placed (`%`) a funnel on a plastic bottle and got this message:

cataclysm__dark_days_ahead_-_0_c-23226-ga2380b5

I was able to press space and continue to play.
The funnel was removed correctly from my inventory and placed on the bottle.

I saved and re-opened the game, everything is ok.

### PR DESCRIPTION
**Pull request by ZehCastro**
More details about this on the bug report. #21586
Should I replicate the info here?

Still learning how to deal with Github, pull-requests, etc.
Please do tell me if something is wrong.

### CODE REVIEW

File: src/item_action.cpp [hunk 1]
```diff
@@ -307,9 +307,7 @@ void game::item_action_menu()
     const item_action_id action = std::get<0>( menu_items[kmenu.ret] );
     item *it = iactions[action];
 
-    if( u.invoke_item( it, action ) ) {
-        u.i_rem( it ); // Need to remove item
-    }
+    u.invoke_item(it, action);
```
Comment (Night-Pryanik) [inline src/item_action.cpp]: Should be styled like `u.invoke_item( it, action );` (with whitespaces inside brackets).

Comment (ZehCastro) [reply src/item_action.cpp]: Will fix my VS options so that i won't mess with style anymore.

**Comment by angoddu** [comment]
Is this PR good to get merged?

**Comment by ZehCastro** [comment]
It probably is good to merge.

It might uncover an item duplication/non-consumption bug somewhere, but the bug should be fixed in the action itself and not in the menu.

**Pull request merged**